### PR TITLE
Project pages

### PR DESCRIPTION
**Pull request by edpaget**
This separates project pages into a new resource speeding up load times for project listings between 3-5x. 

It will also allow us to support custom page types. 

I'm still working on the corresponding Front-End pull request.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -17,6 +17,8 @@
   enable_extension "plpgsql"
   enable_extension "intarray"
   enable_extension "pg_trgm"
+  enable_extension "hstore"
```
Comment (camallen) [inline db/schema.rb]: Seems these extensions have crept in.

File: spec/models/project_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+RSpec.describe ProjectPage, type: :model do
```
Comment (camallen) [inline spec/models/project_page_spec.rb]: doy you want to remove this or leave it pending?

**Comment by camallen** [comment]
LGTM - nice work!

File: db/schema.rb (hunk 1 above)
Comment (edpaget) [inline db/schema.rb]: Will fixed. I wish I could pin down what caused this.

File: spec/models/project_page_spec.rb (hunk 2 above)
Comment (edpaget) [inline spec/models/project_page_spec.rb]: I should probably add some specs.

**Pull request merged**